docs/content/logging-and-monitoring/prometheus-servicemonitor.md

```diff
@@ -0,0 +1,97 @@
+                                ---
+title: Prometheus
+
+description: "Exposing Prometheus metrics using ServiceMonitor."
+weight: 2000
+doctypes: [""]
+aliases:
+- /prometheus-servicemonitor/
+toc: true
+docs: "DOCS-614"
+---
+
+This document demonstrates deploying a Prometheus `ServiceMonitor` resource as part of an NGINX Ingress Controller Helm deployment.
+The `ServiceMonitor` resource will describe the target `Service` to be monitored by a `Prometheus` resource.
+
+# Preface
+This document does **NOT** configure or deploy a `Prometheus` resource or a Prometheus Operator.
+For details on deploying a Prometheus Operator, as well as deploying/configuring a `Prometheus` resource, see the [getting-started](https://prometheus-operator.dev/docs/user-guides/getting-started/) guide in the prometheus-operator docs.
+
+
+# Prerequisites
+Install the latest ServiceMonitor CRD from the [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator) repo:
+```console
+LATEST=$(curl -s https://api.github.com/repos/prometheus-operator/prometheus-operator/releases/latest | jq -cr .tag_name)
+curl https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/$LATEST/example/prometheus-operator-crd/monitoring.coreos.com_servicemonitors.yaml | kubectl create -f -
+```
+
+
+# Install
+To configure a `ServiceMonitor` resource with your Helm to installation, set the following paramaters:
+```shell
+--set prometheus.service.create=true
+--set prometheus.serviceMonitor.create=true
+--set prometheus.serviceMonitor.labels.<your_custom_label_key>="<your_custom_label_value>"
+```
+
+This will deploy:
+1. A `Service` to expose prometheus metrics from the Ingress Controller on port `9113`.
+2. A `ServiceMonitor` resource configured to monitor the service exposing the metrics.
```

Comment:
```suggestion
These changes will:
1. Create a  `Service` to expose Prometheus metrics from NGINX Ingress Controller on port `9113`.
2. Create a `ServiceMonitor` resource configured to monitor the Service exposing the metrics.
```

Reply:
The very first instance of any NGINX product name on any documentation page is supposed to include the F5 prefix for legal branding purposes. I've been lazy about suggesting or enforcing it for a while, but the change is being made across the entire documentation set at the moment.

We also avoid 'the Ingress Controller': in context it's usually "the NGINX Ingress Controller" or "an NGINX Ingress Controller instance".